packages/zmarkdown/package.json

```diff
@@ -90,7 +96,9 @@
     "posttest": "npm run stop",
     "watch": "jest --watch",
     "server": "pm2 start -f server/index.js -i 3 --max-memory-restart 150M",
-    "stop": "pm2 kill"
+    "stop": "pm2 kill",
+    "watch:client": "webpack --watch --info-verbosity verbose --mode development --config default.webpack.config.js",
+    "watch:zhtml": "webpack --watch --info-verbosity verbose --mode development --config zhtml.webpack.config.js"
```

Comment:
* What is the difference between these two?
* Would it make sense to add something for production builds?

Reply:
> What is the difference between these two?

One is just for the wrapper, the other one is for plugins zhtml. If we add zlatex, it we need another entry. But I will investigate the possibility to build all with only one file.

> Would it make sense to add something for production builds?

Of course, I am not adding this for now, but it is planned ;)